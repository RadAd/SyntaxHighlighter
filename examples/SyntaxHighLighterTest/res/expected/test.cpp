comments 0:24 // a single line comment
keyword 28:32 void
comments 48:79 /* a multi
    line comment */
keyword 87:91 void
keyword 107:111 void
comments 124:149 // After void function();
keyword 153:157 void
comments 170:198 /* After void function(); */
keyword 199:203 void
color1 219:222 std
color1 224:230 string
string 238:248 "a string"
string 251:254 'a'
string 263:269 "more"
