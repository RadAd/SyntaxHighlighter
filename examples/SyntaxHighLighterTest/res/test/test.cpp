// a single line comment

void function();

/* a multi
    line comment */
    
void function();

void function(); // After void function();

void function(); /* After void function(); */ void function();

std::string test = "a string" + 'a' + var + "more";
