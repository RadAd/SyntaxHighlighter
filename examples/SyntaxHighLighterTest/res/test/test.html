<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<!-- a single line comment --> Text inbetween <!-- a single line comment -->

<!-- a multi line
	comment --> Text inbetween <!-- a multi
 line comment -->
 
 <h1 var="one" test="two"/>
 