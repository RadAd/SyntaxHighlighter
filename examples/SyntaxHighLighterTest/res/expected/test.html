preprocessor 0:62 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
preprocessor 66:96 <!-- a single line comment -->
preprocessor 112:142 <!-- a single line comment -->
preprocessor 146:177 <!-- a multi line
	comment -->
preprocessor 193:224 <!-- a multi
 line comment -->
keyword 231:233 h1
variable 234:237 var
string 238:243 "one"
variable 244:248 test
string 249:254 "two"
